### PR TITLE
Delete multiple monitor mnrva 230

### PR DESCRIPTION
**Pull request by lovedeep92**
**JIRA:**
See ticket here - https://jira.rax.io/browse/MNRVA-230

**Description:**

1) Create new modal for showing success message If monitor is deleted and error if 404 and other issue occurred during deleting monitor.

Use promiseAll for deleting multiple monitors and check which one failed and which on being succeeded.

**Testing:**
`
npm run test:unit`

 ## Screenshots:
(if applicable)

### CODE REVIEW

File: src/app/_features/monitors/components/list/monitorslist.component.ts [hunk 1]
```diff
@@ -148,4 +161,61 @@ export class MonitorslistComponent implements OnInit {
...
+   * confirmMessageError and confirmMessageSuccess fields are showing success and error messages.
+   * 
+   */
+
+  triggerConfirm() {
+    this.selectedMonForDeletion = [];
+    this.disableOk              = true;
+    this.delMonitor.nativeElement.click();
+    let d = 0;
+    this.selectedMonitors.forEach((element, index) => {
```
Comment (pras7430) [inline src/app/_features/monitors/components/list/monitorslist.component.ts:201]: can we use the index in place of let d=0?

File: src/app/_features/monitors/components/list/monitorslist.component.html [hunk 2]
```diff
@@ -1,18 +1,46 @@
...
+        <hx-progress value="{{progressVal}}"></hx-progress>
+      </hx-div>
+    </div>
+      <div *ngFor="let selectedMonitor of selectedMonForDeletion">
+      <hx-div [ngClass]="selectedMonitor?.error ? 'hxDivErr' : 'hxDivSuccess'">
+        {{selectedMonitor.id}}
+      </hx-div>
+      </div> 
+    <footer>
+      <button class="hxBtn hxPrimary"(click)="triggerOk()" [disabled]="disableOk">Ok</button>
```
Comment (pras7430) [inline src/app/_features/monitors/components/list/monitorslist.component.html:39]: add id for all the buttons

File: src/app/_features/monitors/components/list/monitorslist.component.spec.ts [hunk 3]
```diff
@@ -142,4 +146,60 @@ describe('MonitorslistComponent', () => {
...
+      {id: "afa94898-ace6-42fa-9eb1-d9d17e5261b2", name: "lovedeep-2", labelSelector: {env: "stage"}, labelSelectorMethod: "AND", resourceId: null, summary:{}},
+    ];
+    let spy = spyOn(monitorService, 'deleteMonitorPromise').and.returnValue(new Promise(reject => { reject(new Error('Not found'))}));
+    component.triggerConfirm();
+    expect(spy).toHaveBeenCalledTimes(2);
+  });
+
+  it('should execute progress bar for success', () => {
+    let obj = [{id:"af21671b-2663-4035-810c-8eec9991ca4c", error: true}];       
+    let count;
```
Comment (pras7430) [inline src/app/_features/monitors/components/list/monitorslist.component.spec.ts:186]: one time use variable so can pass value directly

**Review by pras7430 — APPROVED** [review]

File: src/app/_features/monitors/components/list/monitorslist.component.ts (hunk 1 above)
Comment (lovedeep92) [reply src/app/_features/monitors/components/list/monitorslist.component.ts:201]: yes we can .. its done !

File: src/app/_features/monitors/components/list/monitorslist.component.html (hunk 2 above)
Comment (lovedeep92) [reply src/app/_features/monitors/components/list/monitorslist.component.html:39]: done !!

File: src/app/_features/monitors/components/list/monitorslist.component.spec.ts (hunk 3 above)
Comment (lovedeep92) [reply src/app/_features/monitors/components/list/monitorslist.component.spec.ts:186]: I think this should be more readable and looks fine to me!

**Pull request merged**